dev/com.ibm.ws.security.oauth/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -344,6 +344,9 @@ internalClientId.desc=The ID of an existing client that will be used to generate
 internalClientSecret=Client secret used to generate tokens
 internalClientSecret.desc=The secret of an existing client that will be used to generate tokens for PersonalTokenManagement and UsersTokenManagement UI pages.
 
+ropcPreferUserSecurityName=Resource Owner Password Credentials Grant (ROPC) Prefers User Security Name
```

Comment:
Change to:
"Resource Owner Password Credentials (ROPC) grant prefers user security name"

Reply:
done